src/main/java/racingcar/domain/Cars.java

```diff
@@ -0,0 +1,50 @@
+package racingcar.domain;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.stream.Collectors;
+import racingcar.domain.power.PowerGenerator;
+
+public class Cars {
+
+    private final List<Car> cars;
+
+    public Cars(List<String> names) {
+        duplicateValidationName(names);
+
+        this.cars = names.stream()
+                .map(Name::new)
+                .map(Car::new)
+                .toList();
+    }
+
+    private void duplicateValidationName(List<String> name) {
+        if (new HashSet<>(name).size() != name.size()) {
+            throw new IllegalArgumentException("자동차 이름은 중복될 수 없습니다.");
+        }
+    }
```

Comment:
```suggestion
    private void duplicateValidationName(List<String> names) {
        if (new HashSet<>(names).size() != names.size()) {
            throw new IllegalArgumentException("자동차 이름은 중복될 수 없습니다.");
        }
    }
```

리스트 객체의 특성상, 변수명을 복수형으로 작성해주시면 더 명확하게 전달될 것 같아요!

Reply:
아이고 다른 부분에서는 고쳤는데 여기를 못고쳤네요! 리마인드 감사합니다!